WordPress/Classes/ViewRelated/Comments/CommentViewController.m

```diff
@@ -17,28 +17,49 @@
 #import "SuggestionsTableView.h"
 #import "SuggestionService.h"
 
-static NSString *const CVCReplyToastImage = @"action-icon-replied";
-static NSString *const CVCSuccessToastImage = @"action-icon-success";
-static NSString *const CVCHeaderCellIdentifier = @"CommentTableViewHeaderCell";
-static NSString *const CVCCommentCellIdentifier = @"CommentTableViewCell";
-static CGFloat const CVCFirstSectionHeaderHeight = 40;
-static CGFloat const CVCSectionSeparatorHeight = 10;
-static NSInteger const CVCHeaderSectionIndex = 0;
-static NSInteger const CVCNumberOfRows = 1;
-static NSInteger const CVCNumberOfSections = 2;
+
+
+#pragma mark ==========================================================================================
+#pragma mark Constants
+#pragma mark ==========================================================================================
+
+static NSInteger const CommentsDetailsNumberOfSections  = 1;
+static NSInteger const CommentsDetailsHiddenRowNumber   = -1;
+
+typedef NS_ENUM(NSUInteger, CommentsDetailsRow) {
+    CommentsDetailsRowHeader    = 0,
+    CommentsDetailsRowText      = 1,
+    CommentsDetailsRowActions   = 2,
+    CommentsDetailsRowCount     = 3
```

Comment:
Looks like if the enum is updated `CommentsDetailsRowCount` should always be the last item in the enum.  Maybe add a comment about this?